### PR TITLE
New package: teckit-2.5.8

### PR DESCRIPTION
**Pull request by svenper**

### CODE REVIEW

File: srcpkgs/teckit/template [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Template file for 'teckit'
+pkgname=teckit
+version=2.5.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool autoconf automake"
+makedepends="zlib-devel"
+depends="libteckit"
+short_desc="A Text Encoding Conversion toolkit"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: remove article

File: srcpkgs/teckit/template [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+maintainer="svenper <svenper@tuta.io>"
+license="custom"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: find true license

File: srcpkgs/teckit/template [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	}
+}
+
+libteckit-devel_package() {
+	depends="libteckit>=${version}_${revision}"
+	short_desc+=" - library development files"
+	pkg_install() {
+		vmove "usr/include/teckit"
+		vmove "usr/lib/pkgconfig/teckit.pc"
+		vmove "usr/lib/libTECkit.la"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: we do not want libtool files without good reason

File: srcpkgs/teckit/template [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		vmove "usr/lib/libTECkit.so.*"
+		vmove "usr/lib/libTECkit_Compiler.so.*"
+	}
+}
+
+libteckit-devel_package() {
+	depends="libteckit>=${version}_${revision}"
+	short_desc+=" - library development files"
+	pkg_install() {
+		vmove "usr/include/teckit"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: usr/include

File: srcpkgs/teckit/template [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		vmove "usr/lib/pkgconfig/teckit.pc"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: usr/lib/pkgconfig

File: srcpkgs/teckit/template [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+		vmove "usr/lib/libTECkit.la"
+		vmove "usr/lib/libTECkit.so"
+		vmove "usr/lib/libTECkit_Compiler.la"
+		vmove "usr/lib/libTECkit_Compiler.so"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: "usr/lib/*.so"

File: srcpkgs/teckit/template [hunk 7]
```diff
@@ -0,0 +1,44 @@
+# Template file for 'teckit'
+pkgname=teckit
+version=2.5.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool autoconf automake"
+makedepends="zlib-devel"
+depends="libteckit"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: dependency should be found automatically

File: srcpkgs/teckit/template (hunk 2 above)
Comment (svenper) [reply srcpkgs/teckit/template]: it uses *either* CPL or GPL, should they be separated by a comma?

File: srcpkgs/teckit/template (hunk 3 above)
Comment (svenper) [reply srcpkgs/teckit/template]: oops, `.a`, not `.la`

**Comment by maxice8** [comment]
On Mon, Feb 11, 2019 at 05:17:31AM -0800, svenper wrote:
> svenper commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,44 @@
> +# Template file for 'teckit'
> +pkgname=teckit
> +version=2.5.8
> +revision=1
> +build_style=gnu-configure
> +hostmakedepends="libtool autoconf automake"
> +makedepends="zlib-devel"
> +depends="libteckit"
> +short_desc="A Text Encoding Conversion toolkit"
> +maintainer="svenper <svenper@tuta.io>"
> +license="custom"
> 
> it uses *either* CPL or GPL, should they be separated by a comma?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8517#discussion_r255501741

Dual-licensed ? pick one as you wish, preferably the most permissive or the most used.

**Comment by svenper** [comment]
@maxice8 I changed according to your suggestions:

```diff
--- a/srcpkgs/teckit/template
+++ b/srcpkgs/teckit/template
@@ -5,10 +5,9 @@ revision=1
 build_style=gnu-configure
 hostmakedepends="libtool autoconf automake"
 makedepends="zlib-devel"
-depends="libteckit"
-short_desc="A Text Encoding Conversion toolkit"
+short_desc="Text Encoding Conversion toolkit"
 maintainer="svenper <svenper@tuta.io>"
-license="custom"
+license="GPL-2.1-or-later"
 homepage="http://scripts.sil.org/TECkitIntro"
 changelog="http://scripts.sil.org/TECkitRecentChanges"
 distfiles="https://github.com/silnrsi/teckit/archive/v${version}.tar.gz"
@@ -34,11 +33,9 @@ libteckit-devel_package() {
 	depends="libteckit>=${version}_${revision}"
 	short_desc+=" - library development files"
 	pkg_install() {
-		vmove "usr/include/teckit"
-		vmove "usr/lib/pkgconfig/teckit.pc"
-		vmove "usr/lib/libTECkit.la"
-		vmove "usr/lib/libTECkit.so"
-		vmove "usr/lib/libTECkit_Compiler.la"
-		vmove "usr/lib/libTECkit_Compiler.so"
+		vmove "usr/include"
+		vmove "usr/lib/pkgconfig"
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
 	}
 }
```

File: srcpkgs/teckit/template [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+		vmove "usr/lib/libTECkit.so.*"
+		vmove "usr/lib/libTECkit_Compiler.so.*"
+	}
+}
+
+libteckit-devel_package() {
+	depends="libteckit>=${version}_${revision}"
+	short_desc+=" - library development files"
+	pkg_install() {
+		vmove "usr/include"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: unquote

File: srcpkgs/teckit/template [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+		vmove "usr/lib/pkgconfig"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: unquote

File: srcpkgs/teckit/template [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+libteckit_package() {
+	short_desc+=" - library"
+	pkg_install() {
+		vmove "usr/lib/libTECkit.so.*"
+		vmove "usr/lib/libTECkit_Compiler.so.*"
+	}
+}
+
+libteckit-devel_package() {
+	depends="libteckit>=${version}_${revision}"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: please use exact operator `-` not this version or later `>=`

File: srcpkgs/teckit/template [hunk 11]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'teckit'
+pkgname=teckit
+version=2.5.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool autoconf automake"
+makedepends="zlib-devel"
+short_desc="Text Encoding Conversion toolkit"
+maintainer="svenper <svenper@tuta.io>"
+license="GPL-2.1-or-later"
```
Comment (maxice8) [inline srcpkgs/teckit/template]: GPL-2.1-or-later is not a license that exists, do you mean LGPL-2.1-or-later ?

File: srcpkgs/teckit/template [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+changelog="http://scripts.sil.org/TECkitRecentChanges"
+distfiles="https://github.com/silnrsi/teckit/archive/v${version}.tar.gz"
+checksum=0ea52b1304f430aaebff99fec355bc7e4ad75b16ba58959fae5079627f925f93
+
+pre_configure() {
+	./autogen.sh
+}
+
+post_install() {
+	vlicense license/LICENSING.txt
```
Comment (maxice8) [inline srcpkgs/teckit/template]: LGPL-2.1-or-later doesn't need vlicense

**Pull request merged**